### ISSUE TITLE
Replace deprecated library optparse with argparse

### ISSUE THREAD

**Issue by mjtrangoni** [warpnet/salt-lint#128]
Hi @roaldnefs,

As pylint already reported, we are using the **optparse** library which is deprecated, see [here](https://docs.python.org/2/library/optparse.html).

Pylint says,
```bash
saltlint/cli.py:7:0: W0402: Uses of a deprecated module 'optparse' (deprecated-module)
```

As a replacement we have to switch to [argparse](https://docs.python.org/3.7/library/argparse.html#module-argparse).

**Comment by roaldnefs** [comment]
Thanks @mjtrangoni!

We might even contribute this back to `ansible-lint`, because they are also still using `optparse`, see: https://github.com/ansible/ansible-lint/blob/ea4cd39bbcc508a96c17d9e97f12278961023487/lib/ansiblelint/__main__.py#L26.

**Comment by roaldnefs** [comment]
Fixed in version `v0.4.0`.
